koocook_core/models/recipe.py

```diff
@@ -56,7 +57,39 @@ def total_time(self):
 
     @property
     def nutrition(self):
-        return
+        nutrition_list = []
+        for ingredient in self.recipe_ingredients:
+            for nutrient in ingredient.nutrition:
+                if nutrient['nutrient'] not in list(map(lambda x: x['nutrient'], nutrition_list)):
```

Comment:
Prefer using named functions instead of `lambda`s with `map`. An easy way is to use `operator.itemgetter`

```python
import operator
...
map(operator.itemgetter('nutrient'), nutrition_list)
```